@@
-WARNING:  DISTRIBUTED BY clause has no effect in singlenode mode
@@
-WARNING:  DISTRIBUTED BY clause has no effect in singlenode mode
@@
-WARNING:  DISTRIBUTED BY clause has no effect in singlenode mode
@@
-WARNING:  DISTRIBUTED BY clause has no effect in singlenode mode
@@
-
-select count(*) from aocs1;
-  count  
----------
- 1200000
-(1 row)
-
--- test ignorance of `scatter by`
-CREATE TABLE example(a int, b text) DISTRIBUTED by (a);
-WARNING:  DISTRIBUTED BY clause has no effect in singlenode mode
-COPY example FROM stdin;
-CREATE FUNCTION multiset_func(a anytable) RETURNS SETOF example
-    AS '/cb/pgregress/regress.so', 'multiset_example' LANGUAGE C READS SQL DATA;
-explain SELECT * FROM multiset_func( TABLE( SELECT * FROM example ORDER BY a, b SCATTER by a) )  order by a, b;
-WARNING:  SCATTER BY clause has no effect in singlenode mode
-QUERY PLAN
-___________
@@
-SELECT * FROM multiset_func( TABLE( SELECT * FROM example ORDER BY a, b SCATTER by a) )  order by a, b;
-WARNING:  SCATTER BY clause has no effect in singlenode mode
- a |      b      
----+-------------
- 1 |  value1.1/4
- 1 |  value1.2/4
- 1 |  value1.3/4
- 1 |  value1.4/4
- 2 |  value2.1/3
- 2 |  value2.2/3
- 2 |  value2.3/3
- 3 |  value3.1/2
- 3 |  value3.2/2
- 4 |  value4.1/1
-(10 rows)
+select count(*) from aocs1;
+  count  
+---------
+ 1200000
+(1 row)
@@
+-- test ignorance of `scatter by`
+CREATE TABLE example(a int, b text) DISTRIBUTED by (a);
+COPY example FROM stdin;
+CREATE FUNCTION multiset_func(a anytable) RETURNS SETOF example
+    AS '/cb/pgregress/regress.so', 'multiset_example' LANGUAGE C READS SQL DATA;
+ERROR:  type anytable does not exist
+LINE 1: CREATE FUNCTION multiset_func(a anytable) RETURNS SETOF exam...
+                                        ^
+explain SELECT * FROM multiset_func( TABLE( SELECT * FROM example ORDER BY a, b SCATTER by a) )  order by a, b;
+ERROR:  syntax error at or near "TABLE"
+LINE 1: explain SELECT * FROM multiset_func( TABLE( SELECT * FROM ex...
+                                             ^
+SELECT * FROM multiset_func( TABLE( SELECT * FROM example ORDER BY a, b SCATTER by a) )  order by a, b;
+ERROR:  syntax error at or near "TABLE"
+LINE 1: SELECT * FROM multiset_func( TABLE( SELECT * FROM example OR...
+                                     ^
@@
+ERROR:  could not find a function named "multiset_func"
